### PR TITLE
Build docker image, push to quay

### PR DESCRIPTION
**Pull request by DamianSzymanski**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,25 @@
+FROM ruby:2.6.3-slim-buster
+
+LABEL maintainer Travis CI GmbH <support+travis-listener-docker-images@travis-ci.com>
+
+# required for
```
Comment (Duologic) [inline Dockerfile]: Expand comment to 'required for build'?

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,39 @@
+SHELL := bash
+
+VERSION_VALUE ?= $(shell git describe --always --dirty --tags 2>/dev/null)
+DOCKER_IMAGE_REPO ?= travisci/travis-listener
+DOCKER_DEST ?= $(DOCKER_IMAGE_REPO):$(VERSION_VALUE)
+QUAY ?= quay.io
+QUAY_IMAGE ?= $(QUAY)/$(DOCKER_IMAGE_REPO)
```
Comment (Duologic) [inline Makefile]: I'm not opposed to Quay, but why? Let's discuss on Slack.

File: Gemfile [hunk 3]
```diff
@@ -23,6 +23,8 @@ gem 'yajl-ruby',       '~> 1.4.0'
 
 gem 'jemalloc',        git: 'https://github.com/joshk/jemalloc-rb'
 
+gem 'irb', require: false
```
Comment (Duologic) [inline Gemfile]: What does `irb` do? Maybe put it in the commit message so we know why it is here?

File: Makefile [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+.PHONY: docker-build
+docker-build:
+	$(DOCKER) build -t $(DOCKER_DEST) .
+
+.PHONY: docker-push
+docker-push:
+	$(DOCKER) login -u=$(QUAY_ROBOT_HANDLE) -p=$(QUAY_ROBOT_TOKEN) $(QUAY)
+	$(DOCKER) tag $(DOCKER_DEST) $(QUAY_IMAGE):$(VERSION_VALUE)
+	$(DOCKER) push $(QUAY_IMAGE):$(VERSION_VALUE)
```
Comment (JaroslawSokol) [inline Makefile]: I propose to set additional latest tag for master branch:
`ifeq ($(TRAVIS_BRANCH), master)
	$(DOCKER) tag $(DOCKER_DEST) $(QUAY_IMAGE):latest
	$(DOCKER) push $(QUAY_IMAGE):latest
endif`

Comment (DamianSzymanski) [reply Makefile]: @JaroslawSokol I added push for master breanch

**Pull request merged**